docs/team/kvignesh122.md

```diff
@@ -0,0 +1,62 @@
+- Implemented the find feature, which allows users to search for 
+instruments saved in the watchlist using their input search term.
+- Was instrumental in beautifying the program's UI. Coded the ACSII patterns
+for bye, and mTracker logo at startup.
+- Made sure that error messages were consistent in formatting with fullstops 
+and commas. Also made sure grammar is correct for the error messages and the UG due to 
+  my strength in language.
+- Coded the main run method in MTracker class and also the bye command for exiting
+program.
+- Abstracted out IndexedCommand class to handle index input by user, and
+managing it alongside view, delete, done, and edit command classes.
+- Abstracted out AddInstrumentParserTest class for all other instruments' addition parser test classes
+to depend on.
+- Improved overall code quality by abstracting out repetitive codeblocks as
+  methods, and removed redundant setters and getters.
+  
+### Code contributed
+
+[Vignesh's tP Code Dashboard](https://nus-cs2113-ay2122s1.github.io/tp-dashboard/?search=vignesh&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=2021-09-25&tabOpen=true&tabType=authorship&tabAuthor=KVignesh122&tabRepo=AY2122S1-CS2113T-T12-1%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code~other&authorshipIsBinaryFileTypeChecked=false)
+
+### Enhancements
+- Improved user-friendliness and aesthetics of program by
+  making list command to display only the 3 main parameters for all instruments
+  in the watchlist. Implemented a separate View feature for user to 
+  take deeper look into any one of the stored financial instruments.
+- Implemented abort feature in Add and Edit operations for user to exit
+the process of adding/editing an instrument prematurely.
+- Wrote extensive Junit tests for testing abort and addition of instrument features
+for crypto and forex.
+  
+### Documentation:
+- Reviewed grammar and sentence structure of all UG and DG content; made edits to enhance clarity.
+- Contributed UML class diagrams for Command and TextUi sections.
+- Copied over relevant user stories to DG.
+- Wrote more than 60% of the UG. ([#88](https://github.com/AY2122S1-CS2113T-T12-1/tp/pull/88))
+- Wrote Architecture section in DG.
+
+### Team contributions:
+- Major PRs reviewed: [#211](https://github.com/AY2122S1-CS2113T-T12-1/tp/pull/211) 
+  [#95](https://github.com/AY2122S1-CS2113T-T12-1/tp/pull/95)
+  [#91](https://github.com/AY2122S1-CS2113T-T12-1/tp/pull/91)
+  [#83](https://github.com/AY2122S1-CS2113T-T12-1/tp/pull/83)
+- Scheduled and conducted weekly project meetings outside of tutorial/lecture time.
+- Assisted team members with Gradle setup.
+- Planed and maintained team's GitHub issue tracker: [AY2122S1-T12-1](https://github.com/AY2122S1-CS2113T-T12-1/tp/issues)
```

Comment:
typo with 'planed'. Can try to link to specific issues that you opened up. I think that would be good.